### DISCUSSION TITLE
Manuals of the ClassQuiz

### DISCUSSION THREAD

**Discussion by dtusci** [mawoka-myblock/ClassQuiz#286] [General]
Hey guys !

I am trying to understand how to design quizzes using the classquiz.de and then I do not know if you have the manuals to understand how multiple choice, text, order, etc., questions can be structure and the logical way.

Many thanks !

**Comment by TheNomad11** [comment]
How do I mark the correct answers when I create a quiz? Not obvious. Looks like a good quiz, but there is no documentation

**Reply by mawoka-myblock** [reply]
This can be done by pressing the checkmark next to the question.

**Reply by TheNomad11** [reply]
Thanks a lot, there is no checkmark next to the question but there are two checkboxes next to each answer.  The checkmarks clsest to the answer is the one to press, the other checkmark deletes the answer. 
![screen-2024-01-05-19-21-38](https://github.com/mawoka-myblock/ClassQuiz/assets/20747629/025df87f-2d21-4f21-8a95-3c9e47b424c8)

**Comment by TheNomad11** [comment]
Another question. Question type Text. I do not understand that I need to provide at least two correct answers while there is only one correct answer. In "Use case" you write "Test if a name is remembered correctly". Can you explain or do I misunderstand the question type? In my case here is a screenshot regarding German grammar
![screen-2024-01-06-10-50-09](https://github.com/mawoka-myblock/ClassQuiz/assets/20747629/72cda244-ae4c-4e74-9362-ebcf6a4c39ac)

**Reply by mawoka-myblock** [reply]
That's a bug. One answer should be sufficient

**Reply by TheNomad11** [reply]
aha, good to know!

**Reply by TheNomad11** [reply]
> That's a bug. One answer should be sufficient

Sorry for bothering you with another noob question @mawoka-myblock ! It seems that you have fixed the bug immediately according to the repo. Thanks a lot. But it seems I cannot update the usual way? I did what I normally do to update which is docker compose pull and then docker compose up -d, but there were no updates to be pulled. Then I realised that the docker compose file does not look as most others compose files, so I suppose there are more steps involed? Can you help me with this? This should also be something to be added to the docs, I think. Thanks a lot!

**Reply by mawoka-myblock** [reply]
I think that you need to run `docker compose build ` before the up. That should fix it.

**Reply by mawoka-myblock** [reply]
Oh, and no worries for asking. I am happy to help!

**Reply by TheNomad11** [reply]
thanks @mawoka-myblock but there was no change. I did first `docker compose pull`, got the message

```
✔ worker Skipped - No image to be pulled                                                                                                   0.0s 
✔ frontend Skipped - No image to be pulled                                                                                                 0.0s 
✔ api Skipped - No image to be pulled                                                                                                      0.0s 
✔ proxy Pulled                                                                                                                             3.2s 
✔ meilisearch Pulled                                                                                                                       3.2s 
✔ db Pulled                                                                                                                                3.1s 
✔ redis Pulled  
``` 

then `docker compose build` which took a while, and then `docker compose up -d`

The result is the same, "You need at least two answers"

**Reply by mawoka-myblock** [reply]
Hmmm. Gonna check again, but I won't be able to do that before Friday

**Reply by TheNomad11** [reply]
Thanks! Same issue with classquiz.de - still need two answers there

**Reply by TheNomad11** [reply]
I found it out @mawoka-myblock  Before doing anything with docker compose one has to run  `git pull` in the classquiz directory, then you will get the updated version

**Reply by mawoka-myblock** [reply]
Oh, sure! This is a step that was clear in my head! Sorry for not mentioning that and for trying around! I should have mentioned that before.

**Reply by TheNomad11** [reply]
No problem, learning by doing. This is a common issue in tutorials as well, for you experts many things are too obvoius to mention. That's normal.

**Comment by TheNomad11** [comment]
Another question @mawoka-myblock  What is Quiztivity and how to use it? Did not find any documentation and trial and error did not help this time. Thanks again.

**Reply by mawoka-myblock** [reply]
Ah, yes: Quiztivities: Not finished yet. This should provide an alternative to Nearpod when it's finished. Right now, you can create such a quiztivity and share the link with students for example so they can learn something interactively on their own.

**Reply by TheNomad11** [reply]
sounds good! I was not able to create it, though, that means there is no way to save it. But anyway, thanks for your work on it!

**Comment by TheNomad11** [comment]
Another question to you @mawoka-myblock  I cannot find a way to manage users: I jregistered a test user but from my main account I cannot manage users. Thanks for your time and help!

**Reply by mawoka-myblock** [reply]
This isn't possible, but you can manage quizzes at [YOUR_HOST]/moderation

**Reply by TheNomad11** [reply]
hm, okay, thanks
